nitro-protocol/package-lock.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "@statechannels/nitro-protocol",
-  "version": "2.0.0-alpha.2",
+  "version": "2.0.0-alpha.3",
```

Comment:
Any clue why there were some many deletions from this file? Looks like maybe we hadn't run `npm install` recently or something...

Reply:
Not really - I think we've been running `npm ci` as standard practice for a while, so maybe some dependencies up the tree have pruned out their own dependency lists in the meantime?

One expects the opposite to happen on average though.